main.go

```diff
@@ -40,6 +44,30 @@ func main() {
 		log.Fatal("Start MySQL Server!")
 	}
 
+	// Read redis connection/auth info out of REDIS_URL env var already defined
+	u, err := url.Parse(os.Getenv("REDIS_URL"))
+	if err != nil {
+		log.Fatal("No REDIS_URL env var?")
+	}
+	if u.User == nil {
+		log.Fatal("REDIS_URL improper format?")
```

Comment:
We don't set the user for the production redis URL, so this should not be a strict requirement.